components/src/core/components/ProgressBar/ProgressBar.vue

```diff
@@ -34,8 +36,37 @@ export default defineComponent({
       type: String,
       default: '',
     },
+    progressValue: {
+      type: Number,
+      default: undefined,
+    },
   },
+
+  setup: function (props) {
+    const isValuedProgress = ref<boolean>(false);
+    const progressPercentage = ref<string>('');
+    const init = () => {
+      isValuedProgress.value = props.progressValue != undefined ? true : false;
+
+      if (props.progressValue || props.progressValue === 0){
+        if(props.progressValue < 0 || props.progressValue === 0) {
```

Comment:
Check whether the ES-Lint indicates the issue of not having proper indentation for if blocks